src/wp-includes/media.php

```diff
@@ -5564,6 +5680,110 @@ function wp_get_loading_attr_default( $context ) {
 	return 'lazy';
 }
 
+/**
+ * Get loading optimization attributes.
+ *
+ * @since 6.3.0
+ *
+ * @param string $tag_name The tag name.
+ * @param array[] $attr Array of the attributes for the tag.
+ * @param string $context Context for the element for which the loading optimization attribute is requested.
```

Comment:
```suggestion
 * @param string  $tag_name The tag name.
 * @param array[] $attr     Array of the attributes for the tag.
 * @param string  $context  Context for the element for which the loading optimization attribute is requested.
```
Just some alignment correction.